.github/workflows/compat-check-release-release.yml

```diff
@@ -2,6 +2,8 @@ name: Compat Check w/PL - release/release
 
 on:
   workflow_dispatch:
+  schedule:
+    - cron: "0 23 * * 0-6"  # Run daily at 23pm everyday
```

Comment:
Is this run in the UTC-4 timezone, or another? I see discussions on setting timezones, so I think this must be documented somewhere https://github.com/orgs/community/discussions/13454
Also `23pm -> 11pm or 23:00`